### PR TITLE
Feature/infra 1481 call integration tests

### PR DESCRIPTION
**Pull request by ranrubin**
This will trigger the CircleCi pipeline tests that are being developed by @Louis-Dupont

### CODE REVIEW

File: .github/workflows/integration_tests_rc.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Calling CircleCI job
+        shell: bash
+        run: |
+          curl --request POST \
+            --url https://circleci.com/api/v2/project/gh/Deci-AI/algo-integration-tests/pipeline \
+            --header 'Circle-Token: ${{ secrets.CIRCLE_CI_TOKEN }}' \
+            --header 'content-type: application/json' \
+            --data '{"parameters":{"sg_workflow_sg_version":"${{github.ref_name}}", "sg_workflow_install_from_scratch":false, "sg_workflow_run":true, "sg_workflow_rc":true }}'
```
Comment (shaydeci) [inline .github/workflows/integration_tests_rc.yaml:18]: Why don't we install from scratch for RC ?

**Review by shaydeci — CHANGES_REQUESTED** [review]
1 question inline...

File: .github/workflows/integration_tests_rc.yaml (hunk 1 above)
Comment (ranrubin) [reply .github/workflows/integration_tests_rc.yaml:18]: This is what @Louis-Dupont  wanted. 
Installing from scratch can take 15 minutes, and if it happens a few times a day that could be inconvenient.

Feel free to change it to true :)

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**